pilot/pkg/bootstrap/server.go

```diff
@@ -123,7 +123,6 @@ type Server struct {
 	grpcServer          *grpc.Server
 	secureHTTPServer    *http.Server
 	secureGRPCServer    *grpc.Server
-	secureHTTPServerDNS *http.Server
```

Comment:
You are removing this server, which is orthogonal with the LB issue i think. 

But looks good.

Reply:
it's not orthogonal, I just didn't explain it well. right now we serve http and grpc over the same port. To do that we use the net/http stack instead of the grpc stack. This doesn't support max connection age

Reply:
Yep, then same should apply to `secureHTTPServer `?

Reply:
secureHTTPServer is mostly legacy that will be removed soon so I didn't want to touch it too much but I can if we are ok with this approach and want the same applied there?